third_party/move/move-model/bytecode/src/stackless_bytecode.rs

```diff
@@ -484,6 +484,21 @@ impl Bytecode {
         self.is_conditional_branch() || self.is_unconditional_branch()
     }
 
+    pub fn dests(&self) -> Vec<TempIndex> {
+        match self {
+            Bytecode::Assign(_, dst, _, _) => {
+                vec![*dst]
+            }
+            Bytecode::Load(_, dst, _) => {
+                vec![*dst]
+            }
+            Bytecode::Call(_, dsts, _, _, _) => {
+                dsts.clone()
+            }
+            _ => Vec::new(),
```

Comment:
Difficult to say without doc whether this is the expected behavior.

Reply:
Yes, it is. It should be the same set of locals as the written locals of an instruction in the live var analysis.

Reply:
Unfortunately, with the wildcard (`_`) here, a new instruction could be added later that needs handling here but there will be no warnings about it.  If we cannot add an abstraction that links these use cases with the enum definition you should list all Bytecode values here so there is a chance of catching it.